CAIPs/caip-74.md

```diff
@@ -91,11 +84,11 @@ type SignatureMeta struct {
 }
 ```
 
-This construction allows a dApp to uniformly request a SIWE signature regardless of the user's account nature. The user's wallet determines if it should use an EOA or a contract account.
+This construction allows a dApp to uniformly request a SIWx signature regardless of the user's account nature. 
 
 ### Signature Verification
 
-We reconstruct the EIP4361 payload as follows:
+Signature signing and verification should follow the workflow as specified in the [CAIP-122]() namespaces. For example, for `eip155` chains, we reconstruct the SIWx payload as follows:
```

Comment:
```suggestion
Signature signing and verification should follow the workflow as specified in the [CAIP-122]() namespaces. For example, for `eip155` chains, we reconstruct the SIWx payload as follows, resulting in a message conformant with EIP-4361:
```

Reply:
(just to make it more explicit and grokkable to someone reading this with little to no context)